Nodejs/Product/Analysis/Analysis/JsAnalyzer.cs

```diff
@@ -140,8 +140,13 @@ internal partial class JsAnalyzer : IGroupableAnalysisProject, IDeserializeIniti
             }
 
             var tree = Modules.GetModuleTree(Path.GetDirectoryName(filePath));
-            
+
             tree.DefaultPackage = entryPoint;
+            if (dependencies != null) {
+                foreach (var dependency in dependencies) {
+                    RequireAnalysisUnit.UpdateVisibilities(tree, Modules, new ProjectEntry(this, filePath, null), dependency);
```

Comment:
Yeah, I a tried to reuse a project entry, but it was way too much hassle. Moving it out of the loop.